src/Lecture2.hs

```diff
@@ -191,7 +221,7 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge list1 list2 = sort (list1++list2)
```

Comment:
That's cheating 😅 

But it's the less optimal solution. It works in `O((n + m) log (n + m))` time where the optimal solution works in `O (n + m)` time where `n` and `m` are the sizes of the corresponding lists